app/main.ts

```diff
@@ -17,5 +17,6 @@ new Vue({
     router: new VueRouter({
         mode: 'history',
         routes: routes
-    })
+    }),
+    render: h => h('router-view')

```

Comment:
Use render function for top level so we don't need the template compiler at all.